src/grow_worker/worker.py

```diff
@@ -36,7 +41,7 @@ class EarlySystemExit(Exception):
     ...
 
 
-def run_mesido(input_esdl: str) -> str:
+def run_mesido(input_esdl: str) -> Tuple[Optional[str], List[EsdlMessage]]:
```

Comment:
Return type docstring (also the same for `grow_worker_task()`)

Reply:
updated